{
	"vid": "f01292a0-db3c-11e1-a84b-00e0814cab4e",
	"topic": "django -- multiple vulnerabilities",
	"affects": [
		{
			"name": [
				"py26-django"
			],
			"range": [
				{
					"lt": "1.4.1",
					"ge": "1.4"
				},
				{
					"lt": "1.3.2",
					"ge": "1.3"
				}
			]
		},
		{
			"name": [
				"py27-django"
			],
			"range": [
				{
					"lt": "1.4.1",
					"ge": "1.4"
				},
				{
					"lt": "1.3.2",
					"ge": "1.3"
				}
			]
		},
		{
			"name": [
				"py26-django-devel"
			],
			"range": [
				{
					"lt": "20120731,1"
				}
			]
		},
		{
			"name": [
				"py27-django-devel"
			],
			"range": [
				{
					"lt": "20120731,1"
				}
			]
		}
	],
	"description": {
		"xmlns": "http://www.w3.org/1999/xhtml",
		"text": "\n\t<p>The Django project reports:</p>\n\t<blockquote cite=\"https://www.djangoproject.com/weblog/2012/jul/30/security-releases-issued/\">\n\t  <p>Today the Django team is issuing multiple releases --\n\t    Django 1.3.2 and Django 1.4.1 -- to remedy security issues\n\t    reported to us:</p>\n\t  <ul>\n\t    <li>Cross-site scripting in authentication views</li>\n\t    <li>Denial-of-service in image validation</li>\n\t    <li>Denial-of-service via get_image_dimensions()</li>\n\t  </ul>\n\t  <p>All users are encouraged to upgrade Django immediately.</p>\n\t</blockquote>\n      "
	},
	"references": {
		"url": [
			"https://www.djangoproject.com/weblog/2012/jul/30/security-releases-issued/"
		],
		"cvename": [
			"CVE-2012-3442",
			"CVE-2012-3443",
			"CVE-2012-3444"
		]
	},
	"dates": {
		"discovery": "2012-07-30",
		"entry": "2012-07-31",
		"modified": "2014-04-30"
	}
}
